src/test/integration/oauth-test.js

```diff
@@ -348,12 +348,11 @@ describe('oauth/', function() {
       .set(...headerAuth(jwt));
     expect(res.status).toEqual(200);
 
-    // TODO(hobinjk): filter things instead of blocking #793
     let err = await pFinal(chai.request(server)
       .get(Constants.THINGS_PATH)
       .set('Accept', 'application/json')
       .set(...headerAuth(jwt)));
-    expect(err.response.status).toEqual(401);
+    expect(res.status).toEqual(200);
```

Comment:
This test currently doesn't verify that the filtering is working as planned. It should be more like
```
res = chai.request(server)
  .get(Constants.THINGS_PATH)
  .set(...)
  .set(...);
expect(res.status).toEqual(200);
expect(res.body.length).toEqual(1);
expect(res.body[0].href).toEqual('/things/test-1');
```